CMakeLists.txt

```diff
@@ -453,6 +453,15 @@ if(APPLE)
 		src/platform/macos/utils.h)
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
```

Comment:
Any reason why all of this is configurable?
If someone forks Player they will simply replace the files and not mess with cmake options.

Reply:
There is not really a reason. Options where it is useful is ``PLAYER_JS_BUILD_SHELL`` (To decide if you want .js or .html as output) and ``PLAYER_JS_GAME_URL``.
The others are "for style". But I can get rid of them.

Reply:
Yeah, that is what I figured. They could stay, but I really do not see a use for them.